src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -5260,6 +5260,55 @@ internal sealed partial class LazyDynamicObjectCreationExpression : BaseDynamicO
         public override IObjectOrCollectionInitializerOperation Initializer => SetParentOperation(_lazyInitializer.Value, this);
     }
 
+    internal abstract partial class BaseDynamicCollectionElementInitializerOperation : HasDynamicArgumentsExpression, IDynamicCollectionElementInitializerOperation
```

Comment:
>HasDynamicArgumentsExpression, IDynamicCollectionElementInitializerOperation [](start = 87, length = 76)

It looks like for other interfaces that are implemented by classes inheriting from HasDynamicArgumentsExpression, we have extension methods that provide access to the the things stored in the base. 
For example:
```
        public static RefKind? GetArgumentRefKind(this IDynamicInvocationOperation dynamicOperation, int index)
        {
            if (dynamicOperation == null)
            {
                throw new ArgumentNullException(nameof(dynamicOperation));
            }

            return GetArgumentRefKind((HasDynamicArgumentsExpression)dynamicOperation, index);
        }
```


Are we going to provide similar extension methods for IDynamicCollectionElementInitializerOperation? Is there any useful information to expose? If not, then consider saving some memory, inheriting from Operation directly and defining ```public abstract ImmutableArray<IOperation> Arguments { get; }```` in this class directly.